### PR TITLE
Window JSDoc

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by ctrlaltdavid** [comment]
No QA required because no executable code changes.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/5732/app_12020_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/5732/HighFidelity-Beta-PR12020-e6c40f4aae801c09d3c6b0aec978ce50bcdde5d8.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5732/HighFidelity-Beta-PR12020-e6c40f4aae801c09d3c6b0aec978ce50bcdde5d8.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6040/app_12020_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6040/HighFidelity-Beta-PR12020-673bc7de38d15a83e6a224190b95de60b032f194.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6040/HighFidelity-Beta-PR12020-673bc7de38d15a83e6a224190b95de60b032f194.dmg

File: interface/src/scripting/WindowScriptingInterface.h [hunk 1]
```diff
@@ -356,7 +358,7 @@ public slots:
 
     /**jsdoc
      * Emit a {@link Window.connectionAdded|connectionAdded} or a {@link Window.connectionError|connectionError} signal that
-     * indicates whether or not you successfully made a user connection.
+     * indicates whether or a user connection was successfully made using the Web API.
```
Comment (MiladNazeri) [inline interface/src/scripting/WindowScriptingInterface.h]: I think this is missing a not in "indicated whether or (not) a user connection...}

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6072/app_12020_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6072/HighFidelity-Beta-PR12020-7a971b2d26fab899165b196e407184c8c5b06322.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6072/HighFidelity-Beta-PR12020-7a971b2d26fab899165b196e407184c8c5b06322.dmg

**Pull request merged**